### DISCUSSION TITLE
"Show Webcam in navigation" missing in UI-Settings

### DISCUSSION THREAD

**Discussion by MVerBerkmoes** [mainsail-crew/mainsail#1790] [Help] [answered]
As the title conveys, no way to add working webcam to Dashboard as the "Show Webcam in navigation" is missing in UI-Settings.

Tried different web browers with same result .. (unable to attach pic to this posting)

Mainsail v2.9.1
Mainsail-config: v1.2.1-0-ge57810d5

**Comment by meteyou** [comment]
This point was moved to the "Navigation" navi point in the interface setting:

![image](https://github.com/mainsail-crew/mainsail/assets/8167632/0ccd55cb-feb5-46ee-81b8-918d01ce24b1)

**Comment by MVerBerkmoes** [comment]
Thank you for the quick response and that info, but I don’t have Webcam in that tab either ..

**Comment by meteyou** [comment]
then, you don't have a webcam setup... its only there, when you added one webcam.

**Comment by MVerBerkmoes** [comment]
But I can see it in the Webcams tab and it’s working ..

**Comment by meteyou** [comment]
did you check, if the webcam navipoint is right now at the bottom of the list in the interface settings?

**Comment by meteyou** [comment]
ahh... and is it enabled?

**Comment by MVerBerkmoes** [comment]
Sorry .. not sure what this means .. this is the only cam in the webcam list.  And enabled?  Where can that be found?

 

TY again for your help!

**Comment by meteyou** [comment, marked as answer]
There is a button in the webcam list to enable/disable it.

**Comment by MVerBerkmoes** [comment]
I’m an idiot!  I totally missed that in the Dark Interface.

 

TY so much for your patience!

**Reply by eccentricexit** [reply]
Just passing by to say that you are not an idiot. Same happened to me.

Guess lamp is not an intuitive icon for "enable/disable"
